Cvg clean lobby

dlhoff opened the pull request.

jensvdh commented on Assets/RRAssets/Scripts/RRMain.cs:
```diff
@@ -16,6 +16,7 @@ public class RRMain : MonoBehaviour {
         /// of the script instance when the script instance is being loaded.
         /// </summary>
         void Awake() {
+			Debug.Log("Inside RRMain.cs");
```
Do we still need this as part of this PR?

dlhoff replied:
yes. Please see comments. We should discuss.

jensvdh commented on Assets/Scripts/ConvergeGame/MultiConvergeGame.cs:
```diff
@@ -144,35 +157,48 @@ public class MultiConvergeGame : MonoBehaviour
-        buttonWidth = OppViewWidth > 125 ? 125 : OppViewWidth;
+		OppViewWidth = Math.Min((int)(width * 0.25f - 10), 160);
+		buttonWidth = OppViewWidth;
+        // Debug.Log ("Width / OppViewWidth: " + width + " " + OppViewWidth);
```
We can remove this instead of commenting it out. Since we are using git for tracking history.

dlhoff replied:
Removed.

jensvdh commented on Assets/Scripts/ConvergeGame/MultiConvergeGame.cs:
```diff
@@ -298,14 +337,42 @@ void OnGUI ()
+				}
+					
+
 				// On the multiples of 5 seconds, get the names
```
We can remove the commented out code. Since we are using git for tracking history.

dlhoff replied:
This code is important as it reminds me of the past architecture. I may have to go back to that.

jensvdh commented on Assets/Scripts/ConvergeGame/MultiConvergeGame.cs:
```diff
@@ -488,38 +562,46 @@ void MakeWindow (int id)
+			Debug.Log ("showWinners set to true");
+			showWinners = true;
+		}
+
```
There's a lot of enters here. Let's stick with 1.

dlhoff replied:
done

jensvdh commented on Assets/Scripts/ConvergeGame/MultiConvergeGame.cs:
```diff
@@ -488,38 +562,46 @@ void MakeWindow (int id)
+					GUI.color = Color.red;
+					buttonText = name_otherPlayer + " No Bet";
+				}
+				// Debug.Log ("other player button: " + (bufferBorder + width - 170) + " " + topLeft + " " + buttonWidth);
```
We can remove the commented out code. Since we are using git for tracking history.

dlhoff replied:
done.

jensvdh commented on ProjectSettings/ProjectVersion.txt:
```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 5.3.2f1
+m_EditorVersion: 5.3.5f1
```
So we updated the Unity version for the client?

dlhoff replied:
I said ok to the upgrade and it works fine for me.

dlhoff commented:
Jens,


1. Removing RRMain.cs is part of a bigger cleanup. You cannot remove it alone.

2. It's easier for me to look at the code than go through git history.


Perhaps we should meet after the 11am meeting and agree on the changes, and you can show me how to do them.

Do i do another push to the branch with the fixes?


David



________________________________
From: Jens Vanderhaeghe <notifications@github.com>
Sent: Wednesday, September 28, 2016 10:54 AM
To: worldofbalance/unity-game-client
Cc: David Lloyd Hoff; Author
Subject: Re: [worldofbalance/unity-game-client] Cvg clean lobby (#129)


@jensvdh commented on this pull request.

Please fix my remarks and then we can merge

________________________________

In Assets/RRAssets/Scripts/RRMain.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

> @@ -16,6 +16,7 @@ public class RRMain : MonoBehaviour {
         /// of the script instance when the script instance is being loaded.
         /// </summary>
         void Awake() {
+                       Debug.Log("Inside RRMain.cs");


Do we still need this as part of this PR?

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

>          ftr1 = "";
         ftr1P = ftr1.Length * pixelPerChar;

         // DH change
-        OppViewWidth = (int)(width * 0.20f - 10);
-        Debug.Log ("Width / OppViewWidth: " + width + " " + OppViewWidth);
-        buttonWidth = OppViewWidth > 125 ? 125 : OppViewWidth;
+               OppViewWidth = Math.Min((int)(width * 0.25f - 10), 160);
+               buttonWidth = OppViewWidth;
+        // Debug.Log ("Width / OppViewWidth: " + width + " " + OppViewWidth);


We can remove this instead of commenting it out. Since we are using git for tracking history.

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

> +                             if (timeRemain > 15) {
+                                       alarm10Sec = false;
+                                       alarm5Sec = false;
+                               }
+
+                               if ((timeRemain <= 10) && !alarm10Sec) {
+                                       alarm10Sec = true;
+                                       audio.PlayOneShot ((AudioClip)Resources.Load ("Audio/alarm_10sec"));
+                               }
+
+                               if ((timeRemain <= 5) && !alarm5Sec) {
+                                       alarm5Sec = true;
+                                       audio.PlayOneShot ((AudioClip)Resources.Load ("Audio/alarm_5sec"));
+                               }
+
+
                                // On the multiples of 5 seconds, get the names


We can remove the commented out code. Since we are using git for tracking history.

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

>                       GenerateBarGraph ();
                }

+               if (GUI.Button (new Rect (bufferBorder + 180, height - 30 - bufferBorder, 80, 30), "Winners")) {
+                       Debug.Log ("showWinners set to true");
+                       showWinners = true;
+               }
+


There's a lot of enters here. Let's stick with 1.

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

> -                                     // Debug.Log ("other player button: " + (bufferBorder + width - 170) + " " + topLeft + " " + buttonWidth);
-                                       // Debug.Log ("Button text: " + buttonText);
-                                       if (GUI.Button (new Rect (balanceX, topLeft, buttonWidth, 25), buttonText)) {
-                                               barGraph.setOppName (name_otherPlayer);
-                                               displayOtherGraph ();
-                                       }
-                                       topLeft += buttonStep;
-
+                               if (((short)betStatusList [id_otherPlayer]) == 1) {  // bet placed
+                                       GUI.color = Color.green;
+                                       buttonText = name_otherPlayer + " Made Bet";
+                               } else {  // bet not placed
+                                       GUI.color = Color.red;
+                                       buttonText = name_otherPlayer + " No Bet";
+                               }
+                               // Debug.Log ("other player button: " + (bufferBorder + width - 170) + " " + topLeft + " " + buttonWidth);


We can remove the commented out code. Since we are using git for tracking history.

________________________________

In ProjectSettings/ProjectVersion.txt<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

> @@ -1,2 +1,2 @@
-m_EditorVersion: 5.3.2f1
+m_EditorVersion: 5.3.5f1


So we updated the Unity version for the client?

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AQ6W5kYa8a7JS-hs6-4fDkjFaxNC5wLEks5quqnPgaJpZM4KHEAg>.

dlhoff commented:
You can explore the comments here

https://github.com/worldofbalance/unity-game-client/pull/129

and here

https://github.com/worldofbalance/wob-server/pull/48

You can make the adjustments on the existing branch, and push. and it will automatically update the PR. Then I can merge it.

I’d like to do this before tomorrow if possible, because I need these PR’s approved before I can demo/merge my Lobby changes.

Thanks,

Jens




On 28 September 2016 at 11:01:53, David Lloyd Hoff (dhoff@mail.sfsu.edu<mailto:dhoff@mail.sfsu.edu>) wrote:

Jens,


1. Removing RRMain.cs is part of a bigger cleanup. You cannot remove it alone.

2. It's easier for me to look at the code than go through git history.


Perhaps we should meet after the 11am meeting and agree on the changes, and you can show me how to do them.

Do i do another push to the branch with the fixes?


David



________________________________
From: Jens Vanderhaeghe <notifications@github.com>
Sent: Wednesday, September 28, 2016 10:54 AM
To: worldofbalance/unity-game-client
Cc: David Lloyd Hoff; Author
Subject: Re: [worldofbalance/unity-game-client] Cvg clean lobby (#129)


@jensvdh commented on this pull request.

Please fix my remarks and then we can merge

________________________________

In Assets/RRAssets/Scripts/RRMain.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

> @@ -16,6 +16,7 @@ public class RRMain : MonoBehaviour {
         /// of the script instance when the script instance is being loaded.
         /// </summary>
         void Awake() {
+                   Debug.Log("Inside RRMain.cs");


Do we still need this as part of this PR?

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

>          ftr1 = "";
         ftr1P = ftr1.Length * pixelPerChar;

         // DH change
-        OppViewWidth = (int)(width * 0.20f - 10);
-        Debug.Log ("Width / OppViewWidth: " + width + " " + OppViewWidth);
-        buttonWidth = OppViewWidth > 125 ? 125 : OppViewWidth;
+           OppViewWidth = Math.Min((int)(width * 0.25f - 10), 160);
+           buttonWidth = OppViewWidth;
+        // Debug.Log ("Width / OppViewWidth: " + width + " " + OppViewWidth);


We can remove this instead of commenting it out. Since we are using git for tracking history.

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

> +                              if (timeRemain > 15) {
+                                   alarm10Sec = false;
+                                   alarm5Sec = false;
+                           }
+
+                           if ((timeRemain <= 10) && !alarm10Sec) {
+                                   alarm10Sec = true;
+                                   audio.PlayOneShot ((AudioClip)Resources.Load ("Audio/alarm_10sec"));
+                           }
+
+                           if ((timeRemain <= 5) && !alarm5Sec) {
+                                   alarm5Sec = true;
+                                   audio.PlayOneShot ((AudioClip)Resources.Load ("Audio/alarm_5sec"));
+                           }
+
+
                                // On the multiples of 5 seconds, get the names


We can remove the commented out code. Since we are using git for tracking history.

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

>                    GenerateBarGraph ();
                }

+           if (GUI.Button (new Rect (bufferBorder + 180, height - 30 - bufferBorder, 80, 30), "Winners")) {
+                   Debug.Log ("showWinners set to true");
+                   showWinners = true;
+           }
+


There's a lot of enters here. Let's stick with 1.

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

> -                                  // Debug.Log ("other player button: " + (bufferBorder + width - 170) + " " + topLeft + " " + buttonWidth);
-                                       // Debug.Log ("Button text: " + buttonText);
-                                       if (GUI.Button (new Rect (balanceX, topLeft, buttonWidth, 25), buttonText)) {
-                                               barGraph.setOppName (name_otherPlayer);
-                                               displayOtherGraph ();
-                                       }
-                                       topLeft += buttonStep;
-
+                           if (((short)betStatusList [id_otherPlayer]) == 1) {  // bet placed
+                                   GUI.color = Color.green;
+                                   buttonText = name_otherPlayer + " Made Bet";
+                           } else {  // bet not placed
+                                   GUI.color = Color.red;
+                                   buttonText = name_otherPlayer + " No Bet";
+                           }
+                           // Debug.Log ("other player button: " + (bufferBorder + width - 170) + " " + topLeft + " " + buttonWidth);


We can remove the commented out code. Since we are using git for tracking history.

________________________________

In ProjectSettings/ProjectVersion.txt<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

> @@ -1,2 +1,2 @@
-m_EditorVersion: 5.3.2f1
+m_EditorVersion: 5.3.5f1


So we updated the Unity version for the client?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AQ6W5kYa8a7JS-hs6-4fDkjFaxNC5wLEks5quqnPgaJpZM4KHEAg>.

dlhoff commented:
I did the changes requested to the server.

Now i am working on the client.


________________________________
From: Jens Willy Michel Vanderhaeghe
Sent: Wednesday, September 28, 2016 2:38 PM
To: worldofbalance/unity-game-client; worldofbalance/unity-game-client; David Lloyd Hoff
Cc: Author; David Lloyd Hoff
Subject: Re: [worldofbalance/unity-game-client] Cvg clean lobby (#129)

You can explore the comments here

https://github.com/worldofbalance/unity-game-client/pull/129
[https://avatars3.githubusercontent.com/u/17733350?v=3&s=400]<https://github.com/worldofbalance/unity-game-client/pull/129>

Cvg clean lobby by dlhoff · Pull Request #129 · worldofbalance/unity-game-client<https://github.com/worldofbalance/unity-game-client/pull/129>
github.com
unity-game-client - Unity Client for World of Balance


and here

https://github.com/worldofbalance/wob-server/pull/48
[https://avatars3.githubusercontent.com/u/17733350?v=3&s=400]<https://github.com/worldofbalance/wob-server/pull/48>

UI changes from Sept game sessions by dlhoff · Pull Request #48 · worldofbalance/wob-server<https://github.com/worldofbalance/wob-server/pull/48>
github.com
Contribute to wob-server development by creating an account on GitHub.


You can make the adjustments on the existing branch, and push. and it will automatically update the PR. Then I can merge it.

I’d like to do this before tomorrow if possible, because I need these PR’s approved before I can demo/merge my Lobby changes.

Thanks,

Jens




On 28 September 2016 at 11:01:53, David Lloyd Hoff (dhoff@mail.sfsu.edu<mailto:dhoff@mail.sfsu.edu>) wrote:

Jens,


1. Removing RRMain.cs is part of a bigger cleanup. You cannot remove it alone.

2. It's easier for me to look at the code than go through git history.


Perhaps we should meet after the 11am meeting and agree on the changes, and you can show me how to do them.

Do i do another push to the branch with the fixes?


David



________________________________
From: Jens Vanderhaeghe <notifications@github.com>
Sent: Wednesday, September 28, 2016 10:54 AM
To: worldofbalance/unity-game-client
Cc: David Lloyd Hoff; Author
Subject: Re: [worldofbalance/unity-game-client] Cvg clean lobby (#129)


@jensvdh commented on this pull request.

Please fix my remarks and then we can merge

________________________________

In Assets/RRAssets/Scripts/RRMain.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

> @@ -16,6 +16,7 @@ public class RRMain : MonoBehaviour {
         /// of the script instance when the script instance is being loaded.
         /// </summary>
         void Awake() {
+                   Debug.Log("Inside RRMain.cs");


Do we still need this as part of this PR?

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

>          ftr1 = "";
         ftr1P = ftr1.Length * pixelPerChar;

         // DH change
-        OppViewWidth = (int)(width * 0.20f - 10);
-        Debug.Log ("Width / OppViewWidth: " + width + " " + OppViewWidth);
-        buttonWidth = OppViewWidth > 125 ? 125 : OppViewWidth;
+           OppViewWidth = Math.Min((int)(width * 0.25f - 10), 160);
+           buttonWidth = OppViewWidth;
+        // Debug.Log ("Width / OppViewWidth: " + width + " " + OppViewWidth);


We can remove this instead of commenting it out. Since we are using git for tracking history.

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

> +                              if (timeRemain > 15) {
+                                   alarm10Sec = false;
+                                   alarm5Sec = false;
+                           }
+
+                           if ((timeRemain <= 10) && !alarm10Sec) {
+                                   alarm10Sec = true;
+                                   audio.PlayOneShot ((AudioClip)Resources.Load ("Audio/alarm_10sec"));
+                           }
+
+                           if ((timeRemain <= 5) && !alarm5Sec) {
+                                   alarm5Sec = true;
+                                   audio.PlayOneShot ((AudioClip)Resources.Load ("Audio/alarm_5sec"));
+                           }
+
+
                                // On the multiples of 5 seconds, get the names


We can remove the commented out code. Since we are using git for tracking history.

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

>                    GenerateBarGraph ();
                }

+           if (GUI.Button (new Rect (bufferBorder + 180, height - 30 - bufferBorder, 80, 30), "Winners")) {
+                   Debug.Log ("showWinners set to true");
+                   showWinners = true;
+           }
+


There's a lot of enters here. Let's stick with 1.

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

> -                                  // Debug.Log ("other player button: " + (bufferBorder + width - 170) + " " + topLeft + " " + buttonWidth);
-                                       // Debug.Log ("Button text: " + buttonText);
-                                       if (GUI.Button (new Rect (balanceX, topLeft, buttonWidth, 25), buttonText)) {
-                                               barGraph.setOppName (name_otherPlayer);
-                                               displayOtherGraph ();
-                                       }
-                                       topLeft += buttonStep;
-
+                           if (((short)betStatusList [id_otherPlayer]) == 1) {  // bet placed
+                                   GUI.color = Color.green;
+                                   buttonText = name_otherPlayer + " Made Bet";
+                           } else {  // bet not placed
+                                   GUI.color = Color.red;
+                                   buttonText = name_otherPlayer + " No Bet";
+                           }
+                           // Debug.Log ("other player button: " + (bufferBorder + width - 170) + " " + topLeft + " " + buttonWidth);


We can remove the commented out code. Since we are using git for tracking history.

________________________________

In ProjectSettings/ProjectVersion.txt<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

> @@ -1,2 +1,2 @@
-m_EditorVersion: 5.3.2f1
+m_EditorVersion: 5.3.5f1


So we updated the Unity version for the client?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AQ6W5kYa8a7JS-hs6-4fDkjFaxNC5wLEks5quqnPgaJpZM4KHEAg>.

jensvdh commented:
Thanks, David!


On 28 September 2016 at 14:54:06, dlhoff (notifications@github.com) wrote:

I did the changes requested to the server.

Now i am working on the client.


________________________________
From: Jens Willy Michel Vanderhaeghe
Sent: Wednesday, September 28, 2016 2:38 PM
To: worldofbalance/unity-game-client; worldofbalance/unity-game-client;
David Lloyd Hoff
Cc: Author; David Lloyd Hoff
Subject: Re: [worldofbalance/unity-game-client] Cvg clean lobby (#129)

You can explore the comments here

https://github.com/worldofbalance/unity-game-client/pull/129
[https://avatars3.githubusercontent.com/u/17733350?v=3&s=400]<
https://github.com/worldofbalance/unity-game-client/pull/129>

Cvg clean lobby by dlhoff · Pull Request #129 ·
worldofbalance/unity-game-client<
https://github.com/worldofbalance/unity-game-client/pull/129>
github.com
unity-game-client - Unity Client for World of Balance


and here

https://github.com/worldofbalance/wob-server/pull/48
[https://avatars3.githubusercontent.com/u/17733350?v=3&s=400]<
https://github.com/worldofbalance/wob-server/pull/48>

UI changes from Sept game sessions by dlhoff · Pull Request #48 ·
worldofbalance/wob-server<
https://github.com/worldofbalance/wob-server/pull/48>
github.com
Contribute to wob-server development by creating an account on GitHub.


You can make the adjustments on the existing branch, and push. and it will
automatically update the PR. Then I can merge it.

I’d like to do this before tomorrow if possible, because I need these PR’s
approved before I can demo/merge my Lobby changes.

Thanks,

Jens




On 28 September 2016 at 11:01:53, David Lloyd Hoff (dhoff@mail.sfsu.edu
<mailto:dhoff@mail.sfsu.edu>) wrote:

Jens,


1. Removing RRMain.cs is part of a bigger cleanup. You cannot remove it
alone.

2. It's easier for me to look at the code than go through git history.


Perhaps we should meet after the 11am meeting and agree on the changes, and
you can show me how to do them.

Do i do another push to the branch with the fixes?


David



________________________________
From: Jens Vanderhaeghe <notifications@github.com>
Sent: Wednesday, September 28, 2016 10:54 AM
To: worldofbalance/unity-game-client
Cc: David Lloyd Hoff; Author
Subject: Re: [worldofbalance/unity-game-client] Cvg clean lobby (#129)


@jensvdh commented on this pull request.

Please fix my remarks and then we can merge

________________________________

In Assets/RRAssets/Scripts/RRMain.cs<
https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491
>:

> @@ -16,6 +16,7 @@ public class RRMain : MonoBehaviour {
/// of the script instance when the script instance is being loaded.
/// </summary>
void Awake() {
+ Debug.Log("Inside RRMain.cs");


Do we still need this as part of this PR?

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<
https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491
>:

> ftr1 = "";
ftr1P = ftr1.Length * pixelPerChar;

// DH change
- OppViewWidth = (int)(width * 0.20f - 10);
- Debug.Log ("Width / OppViewWidth: " + width + " " + OppViewWidth);
- buttonWidth = OppViewWidth > 125 ? 125 : OppViewWidth;
+ OppViewWidth = Math.Min((int)(width * 0.25f - 10), 160);
+ buttonWidth = OppViewWidth;
+ // Debug.Log ("Width / OppViewWidth: " + width + " " + OppViewWidth);


We can remove this instead of commenting it out. Since we are using git for
tracking history.

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<
https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491
>:

> + if (timeRemain > 15) {
+ alarm10Sec = false;
+ alarm5Sec = false;
+ }
+
+ if ((timeRemain <= 10) && !alarm10Sec) {
+ alarm10Sec = true;
+ audio.PlayOneShot ((AudioClip)Resources.Load ("Audio/alarm_10sec"));
+ }
+
+ if ((timeRemain <= 5) && !alarm5Sec) {
+ alarm5Sec = true;
+ audio.PlayOneShot ((AudioClip)Resources.Load ("Audio/alarm_5sec"));
+ }
+
+
// On the multiples of 5 seconds, get the names


We can remove the commented out code. Since we are using git for tracking
history.

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<
https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491
>:

> GenerateBarGraph ();
}

+ if (GUI.Button (new Rect (bufferBorder + 180, height - 30 - bufferBorder,
80, 30), "Winners")) {
+ Debug.Log ("showWinners set to true");
+ showWinners = true;
+ }
+


There's a lot of enters here. Let's stick with 1.

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<
https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491
>:

> - // Debug.Log ("other player button: " + (bufferBorder + width - 170) +
" " + topLeft + " " + buttonWidth);
- // Debug.Log ("Button text: " + buttonText);
- if (GUI.Button (new Rect (balanceX, topLeft, buttonWidth, 25),
buttonText)) {
- barGraph.setOppName (name_otherPlayer);
- displayOtherGraph ();
- }
- topLeft += buttonStep;
-
+ if (((short)betStatusList [id_otherPlayer]) == 1) { // bet placed
+ GUI.color = Color.green;
+ buttonText = name_otherPlayer + " Made Bet";
+ } else { // bet not placed
+ GUI.color = Color.red;
+ buttonText = name_otherPlayer + " No Bet";
+ }
+ // Debug.Log ("other player button: " + (bufferBorder + width - 170) + "
" + topLeft + " " + buttonWidth);


We can remove the commented out code. Since we are using git for tracking
history.

________________________________

In ProjectSettings/ProjectVersion.txt<
https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491
>:

> @@ -1,2 +1,2 @@
-m_EditorVersion: 5.3.2f1
+m_EditorVersion: 5.3.5f1


So we updated the Unity version for the client?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<
https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>,
or mute the thread<
https://github.com/notifications/unsubscribe-auth/AQ6W5kYa8a7JS-hs6-4fDkjFaxNC5wLEks5quqnPgaJpZM4KHEAg
>.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/worldofbalance/unity-game-client/pull/129#issuecomment-250312008>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AChKYex_3h7q4WeT0Ev3Yv1-K3At0oK2ks5quuH-gaJpZM4KHEAg>
.

dlhoff commented:
I updated the client as well.

Some comments.

1. We cannot remove RRMain. We can discuss tomorrow.

2. I removed many of the commented-out blocks you requested. I want to keep the block around 358 (new number). It gives me information about the architecture in the past that may be important later. It is much easier if it is in the code in front of me than searching through snippets of git commits.

3. I removed the extra blank lines as you requested. Thanks.

4. I did the upgrade when requested. Seems to work fine. Is that ok?


David



________________________________
From: Jens Willy Michel Vanderhaeghe
Sent: Wednesday, September 28, 2016 2:38 PM
To: worldofbalance/unity-game-client; worldofbalance/unity-game-client; David Lloyd Hoff
Cc: Author; David Lloyd Hoff
Subject: Re: [worldofbalance/unity-game-client] Cvg clean lobby (#129)

You can explore the comments here

https://github.com/worldofbalance/unity-game-client/pull/129
[https://avatars3.githubusercontent.com/u/17733350?v=3&s=400]<https://github.com/worldofbalance/unity-game-client/pull/129>

Cvg clean lobby by dlhoff · Pull Request #129 · worldofbalance/unity-game-client<https://github.com/worldofbalance/unity-game-client/pull/129>
github.com
unity-game-client - Unity Client for World of Balance


and here

https://github.com/worldofbalance/wob-server/pull/48
[https://avatars3.githubusercontent.com/u/17733350?v=3&s=400]<https://github.com/worldofbalance/wob-server/pull/48>

UI changes from Sept game sessions by dlhoff · Pull Request #48 · worldofbalance/wob-server<https://github.com/worldofbalance/wob-server/pull/48>
github.com
Contribute to wob-server development by creating an account on GitHub.


You can make the adjustments on the existing branch, and push. and it will automatically update the PR. Then I can merge it.

I’d like to do this before tomorrow if possible, because I need these PR’s approved before I can demo/merge my Lobby changes.

Thanks,

Jens




On 28 September 2016 at 11:01:53, David Lloyd Hoff (dhoff@mail.sfsu.edu<mailto:dhoff@mail.sfsu.edu>) wrote:

Jens,


1. Removing RRMain.cs is part of a bigger cleanup. You cannot remove it alone.

2. It's easier for me to look at the code than go through git history.


Perhaps we should meet after the 11am meeting and agree on the changes, and you can show me how to do them.

Do i do another push to the branch with the fixes?


David



________________________________
From: Jens Vanderhaeghe <notifications@github.com>
Sent: Wednesday, September 28, 2016 10:54 AM
To: worldofbalance/unity-game-client
Cc: David Lloyd Hoff; Author
Subject: Re: [worldofbalance/unity-game-client] Cvg clean lobby (#129)


@jensvdh commented on this pull request.

Please fix my remarks and then we can merge

________________________________

In Assets/RRAssets/Scripts/RRMain.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

> @@ -16,6 +16,7 @@ public class RRMain : MonoBehaviour {
         /// of the script instance when the script instance is being loaded.
         /// </summary>
         void Awake() {
+                   Debug.Log("Inside RRMain.cs");


Do we still need this as part of this PR?

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

>          ftr1 = "";
         ftr1P = ftr1.Length * pixelPerChar;

         // DH change
-        OppViewWidth = (int)(width * 0.20f - 10);
-        Debug.Log ("Width / OppViewWidth: " + width + " " + OppViewWidth);
-        buttonWidth = OppViewWidth > 125 ? 125 : OppViewWidth;
+           OppViewWidth = Math.Min((int)(width * 0.25f - 10), 160);
+           buttonWidth = OppViewWidth;
+        // Debug.Log ("Width / OppViewWidth: " + width + " " + OppViewWidth);


We can remove this instead of commenting it out. Since we are using git for tracking history.

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

> +                              if (timeRemain > 15) {
+                                   alarm10Sec = false;
+                                   alarm5Sec = false;
+                           }
+
+                           if ((timeRemain <= 10) && !alarm10Sec) {
+                                   alarm10Sec = true;
+                                   audio.PlayOneShot ((AudioClip)Resources.Load ("Audio/alarm_10sec"));
+                           }
+
+                           if ((timeRemain <= 5) && !alarm5Sec) {
+                                   alarm5Sec = true;
+                                   audio.PlayOneShot ((AudioClip)Resources.Load ("Audio/alarm_5sec"));
+                           }
+
+
                                // On the multiples of 5 seconds, get the names


We can remove the commented out code. Since we are using git for tracking history.

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

>                    GenerateBarGraph ();
                }

+           if (GUI.Button (new Rect (bufferBorder + 180, height - 30 - bufferBorder, 80, 30), "Winners")) {
+                   Debug.Log ("showWinners set to true");
+                   showWinners = true;
+           }
+


There's a lot of enters here. Let's stick with 1.

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

> -                                  // Debug.Log ("other player button: " + (bufferBorder + width - 170) + " " + topLeft + " " + buttonWidth);
-                                       // Debug.Log ("Button text: " + buttonText);
-                                       if (GUI.Button (new Rect (balanceX, topLeft, buttonWidth, 25), buttonText)) {
-                                               barGraph.setOppName (name_otherPlayer);
-                                               displayOtherGraph ();
-                                       }
-                                       topLeft += buttonStep;
-
+                           if (((short)betStatusList [id_otherPlayer]) == 1) {  // bet placed
+                                   GUI.color = Color.green;
+                                   buttonText = name_otherPlayer + " Made Bet";
+                           } else {  // bet not placed
+                                   GUI.color = Color.red;
+                                   buttonText = name_otherPlayer + " No Bet";
+                           }
+                           // Debug.Log ("other player button: " + (bufferBorder + width - 170) + " " + topLeft + " " + buttonWidth);


We can remove the commented out code. Since we are using git for tracking history.

________________________________

In ProjectSettings/ProjectVersion.txt<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>:

> @@ -1,2 +1,2 @@
-m_EditorVersion: 5.3.2f1
+m_EditorVersion: 5.3.5f1


So we updated the Unity version for the client?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AQ6W5kYa8a7JS-hs6-4fDkjFaxNC5wLEks5quqnPgaJpZM4KHEAg>.

jensvdh commented:
Yes. Looks good. I’ll go ahead and merge your PR’s.


On 28 September 2016 at 15:08:05, dlhoff (notifications@github.com) wrote:

I updated the client as well.

Some comments.

1. We cannot remove RRMain. We can discuss tomorrow.

2. I removed many of the commented-out blocks you requested. I want to keep
the block around 358 (new number). It gives me information about the
architecture in the past that may be important later. It is much easier if
it is in the code in front of me than searching through snippets of git
commits.

3. I removed the extra blank lines as you requested. Thanks.

4. I did the upgrade when requested. Seems to work fine. Is that ok?


David



________________________________
From: Jens Willy Michel Vanderhaeghe
Sent: Wednesday, September 28, 2016 2:38 PM
To: worldofbalance/unity-game-client; worldofbalance/unity-game-client;
David Lloyd Hoff
Cc: Author; David Lloyd Hoff
Subject: Re: [worldofbalance/unity-game-client] Cvg clean lobby (#129)

You can explore the comments here

https://github.com/worldofbalance/unity-game-client/pull/129
[https://avatars3.githubusercontent.com/u/17733350?v=3&s=400]<
https://github.com/worldofbalance/unity-game-client/pull/129>

Cvg clean lobby by dlhoff · Pull Request #129 ·
worldofbalance/unity-game-client<
https://github.com/worldofbalance/unity-game-client/pull/129>
github.com
unity-game-client - Unity Client for World of Balance


and here

https://github.com/worldofbalance/wob-server/pull/48
[https://avatars3.githubusercontent.com/u/17733350?v=3&s=400]<
https://github.com/worldofbalance/wob-server/pull/48>

UI changes from Sept game sessions by dlhoff · Pull Request #48 ·
worldofbalance/wob-server<
https://github.com/worldofbalance/wob-server/pull/48>
github.com
Contribute to wob-server development by creating an account on GitHub.


You can make the adjustments on the existing branch, and push. and it will
automatically update the PR. Then I can merge it.

I’d like to do this before tomorrow if possible, because I need these PR’s
approved before I can demo/merge my Lobby changes.

Thanks,

Jens




On 28 September 2016 at 11:01:53, David Lloyd Hoff (dhoff@mail.sfsu.edu
<mailto:dhoff@mail.sfsu.edu>) wrote:

Jens,


1. Removing RRMain.cs is part of a bigger cleanup. You cannot remove it
alone.

2. It's easier for me to look at the code than go through git history.


Perhaps we should meet after the 11am meeting and agree on the changes, and
you can show me how to do them.

Do i do another push to the branch with the fixes?


David



________________________________
From: Jens Vanderhaeghe <notifications@github.com>
Sent: Wednesday, September 28, 2016 10:54 AM
To: worldofbalance/unity-game-client
Cc: David Lloyd Hoff; Author
Subject: Re: [worldofbalance/unity-game-client] Cvg clean lobby (#129)


@jensvdh commented on this pull request.

Please fix my remarks and then we can merge

________________________________

In Assets/RRAssets/Scripts/RRMain.cs<
https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491
>:

> @@ -16,6 +16,7 @@ public class RRMain : MonoBehaviour {
/// of the script instance when the script instance is being loaded.
/// </summary>
void Awake() {
+ Debug.Log("Inside RRMain.cs");


Do we still need this as part of this PR?

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<
https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491
>:

> ftr1 = "";
ftr1P = ftr1.Length * pixelPerChar;

// DH change
- OppViewWidth = (int)(width * 0.20f - 10);
- Debug.Log ("Width / OppViewWidth: " + width + " " + OppViewWidth);
- buttonWidth = OppViewWidth > 125 ? 125 : OppViewWidth;
+ OppViewWidth = Math.Min((int)(width * 0.25f - 10), 160);
+ buttonWidth = OppViewWidth;
+ // Debug.Log ("Width / OppViewWidth: " + width + " " + OppViewWidth);


We can remove this instead of commenting it out. Since we are using git for
tracking history.

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<
https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491
>:

> + if (timeRemain > 15) {
+ alarm10Sec = false;
+ alarm5Sec = false;
+ }
+
+ if ((timeRemain <= 10) && !alarm10Sec) {
+ alarm10Sec = true;
+ audio.PlayOneShot ((AudioClip)Resources.Load ("Audio/alarm_10sec"));
+ }
+
+ if ((timeRemain <= 5) && !alarm5Sec) {
+ alarm5Sec = true;
+ audio.PlayOneShot ((AudioClip)Resources.Load ("Audio/alarm_5sec"));
+ }
+
+
// On the multiples of 5 seconds, get the names


We can remove the commented out code. Since we are using git for tracking
history.

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<
https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491
>:

> GenerateBarGraph ();
}

+ if (GUI.Button (new Rect (bufferBorder + 180, height - 30 - bufferBorder,
80, 30), "Winners")) {
+ Debug.Log ("showWinners set to true");
+ showWinners = true;
+ }
+


There's a lot of enters here. Let's stick with 1.

________________________________

In Assets/Scripts/ConvergeGame/MultiConvergeGame.cs<
https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491
>:

> - // Debug.Log ("other player button: " + (bufferBorder + width - 170) +
" " + topLeft + " " + buttonWidth);
- // Debug.Log ("Button text: " + buttonText);
- if (GUI.Button (new Rect (balanceX, topLeft, buttonWidth, 25),
buttonText)) {
- barGraph.setOppName (name_otherPlayer);
- displayOtherGraph ();
- }
- topLeft += buttonStep;
-
+ if (((short)betStatusList [id_otherPlayer]) == 1) { // bet placed
+ GUI.color = Color.green;
+ buttonText = name_otherPlayer + " Made Bet";
+ } else { // bet not placed
+ GUI.color = Color.red;
+ buttonText = name_otherPlayer + " No Bet";
+ }
+ // Debug.Log ("other player button: " + (bufferBorder + width - 170) + "
" + topLeft + " " + buttonWidth);


We can remove the commented out code. Since we are using git for tracking
history.

________________________________

In ProjectSettings/ProjectVersion.txt<
https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491
>:

> @@ -1,2 +1,2 @@
-m_EditorVersion: 5.3.2f1
+m_EditorVersion: 5.3.5f1


So we updated the Unity version for the client?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<
https://github.com/worldofbalance/unity-game-client/pull/129#pullrequestreview-1992491>,
or mute the thread<
https://github.com/notifications/unsubscribe-auth/AQ6W5kYa8a7JS-hs6-4fDkjFaxNC5wLEks5quqnPgaJpZM4KHEAg
>.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/worldofbalance/unity-game-client/pull/129#issuecomment-250316038>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AChKYWsVew89RCNOhpJ3uMM-jojVHOATks5quuVFgaJpZM4KHEAg>
.
